### DISCUSSION TITLE
[Request] Load Ollama Service from external service

### DISCUSSION THREAD

**Discussion by alaahong** [lobehub/lobehub#1467] [Self Hosting] [answered]
### 🥰 Feature Description

当前使用本地Ollama服务的时候，通过docker的host.docker.internal 访问代理，有一定限制，能否提供配置直接调取外部服务？
否则，例如该代理是Docker的概念，使用podman等其他运行时或者K8S环境下，没有思路如何配置外部Ollama地址以供调用。

### 🧐 Proposed Solution

提供外部Ollama服务配置

### 📝 Additional Information

N/A

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
你把 `OLLAMA_PROXY_URL` 改成其他外部服务的 url 就好了吧？

[a comment by an automated account is not shown]

**Comment by alaahong** [comment, marked as answer]
> 你把 `OLLAMA_PROXY_URL` 改成其他外部服务的 url 就好了吧？

是的，开始lobe 跟 ollama在同一个pod启动的，localhost读不到，后来改成service的地址就可以了。 

Thanks

[a comment by an automated account is not shown]
